README.md

```diff
@@ -561,6 +564,24 @@ will do the rest of the work for you.
 
 ## Staging
 
+<a href="#i-want-to-stage-all-tracked-files-and-leave-untracked-files"></a>
+
+### I want to stage all tracked files and leave untracked files
+
+```sh
+(my-branch*)$ git add -u
```

Comment:
Remove `(my-branch*)` here and in the following two commands. It isn't necessary, and it doesn't follow the naming conventions for the rest of the document.

Reply:
done

Reply:
done